android/build.gradle

```diff
@@ -44,7 +44,7 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    api "com.google.android.gms:play-services-cast-framework:19.0.0"
+    api "com.google.android.gms:play-services-cast-framework:21.2.0"
```

Comment:
@gianlucaparadise took a look at upgrading Android version of cast framework, however I'm a bit out of depth with my Kotlin skills!